functions/Get-DbaTable.ps1

```diff
@@ -0,0 +1,187 @@
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.	
+	
+.LINK
+https://dbatools.io/Get-DbaTable
+.EXAMPLE
+Get-DbaTable -SqlInstance DEV01 -Database Test1
+Return all tables in the Test1 database
+.EXAMPLE
+Get-DbaTable -SqlInstance DEV01 -Database MyDB -Table MyTable
+Return only information on the table MyTable from the database MyDB
+.EXAMPLE
+Get-DbaTable -SqlInstance DEV01 -Database MyDB -Table MyTable
+using pipeline
+.EXAMPLE
+Get-DbaTable -SqlInstance DEV01 -Database MyDB -Table MyTable
+with multiple instances
+.EXAMPLE
+Get-DbaTable -SqlInstance DEV01 -Table MyTable
+REturns information on table called MyTable if it exists in any database on the server, under any schema
+
+
+#>
+	[CmdletBinding()]
+	param ([parameter(ValueFromPipeline, Mandatory = $true)]
+		[Alias("ServerInstance", "SqlServer")]
+		[object[]]$SqlInstance,
+		[System.Management.Automation.PSCredential]$SqlCredential,
+		[switch]$IncludeSystemDBs,
+		[string[]]$Table,
+		[switch]$Silent
+	)
+	
+	DynamicParam { if ($SqlInstance) { return Get-ParamSqlDatabases -SqlServer $SqlInstance[0] -SqlCredential $SourceSqlCredential } }
+	
+	BEGIN
+	{
+		$databases = $psboundparameters.Databases
+		$exclude = $psboundparameters.Exclude
+		
+        $defaultprops = @("Parent", "Schema", "Name", "IndexSpaceUsed", "DataSpaceUsed", "RowCount", "HasClusteredIndex", "IsFileTable", "IsMemoryOptimized", "IsPartitioned", "FullTextIndex", "ChangeTrackingEnabled")
+
+		$fqtns = @()
+		
+		if ($Table)
+		{
+			foreach ($t in $Table)
+			{
+				$dotcount = ([regex]::Matches($t, "\.")).count
+				
+                $database = $NULL
+                $Schema = $NULL
+
+				if ($dotcount -eq 1)
+				{
+					$schema = $t.Split(".")[0]
+					$tbl = $t.Split(".")[1]
+				}
+				
+				if ($dotcount -eq 2)
+				{
+					$database = $t.Split(".")[0]
+					$schema = $t.Split(".")[1]
+					$tbl = $t.Split(".")[2]
+				}
+				
+				$fqtn = [PSCustomObject] @{
+					Database = $database
+					Schema = $Schema
+					Table = $tbl
+				}
+				$fqtns += $fqtn
+			}
+		}
+        #$fqtns
+	}
+	
+	PROCESS
+	{
+		foreach ($instance in $sqlinstance)
+		{	
+			try
+			{
+				Write-Message -Level Verbose -Message "Connecting to $instance"
+				$server = Connect-SqlServer -SqlServer $instance -SqlCredential $sqlcredential
+			}
+			catch
+			{
+				Stop-Function -Message "Failed to connect to: $instance" -Continue -Target $instance -InnerErrorRecord $_
+			}
+			
+			#If IncludeSystemDBs is true, include systemdbs
+			#only look at online databases (Status equal normal)
+			try
+			{
+				if ($databases.length -gt 0)
+				{
+					$dbs = $server.Databases | Where-Object { $databases -contains $_.Name }
```

Comment:
sorry @Staggerlee011 , I forgot this in the first review:
shouldn't the check for status -eq 'Normal' also be added when $databases are submitted?